Makefile

```diff
@@ -108,7 +108,6 @@ staticcheck:
 		`go list ./... | grep -v '\/flux\/stdlib\>'`
 	GO111MODULE=on ./gotool.sh honnef.co/go/tools/cmd/staticcheck ./stdlib/...
 
-test: test-go test-rust test-flux test-wasm
```

Comment:
We don't want to remove this line entirely. This is what CI uses to run tests. We just want to remove `test-wasm` from the end of this line.